frontend/src/reducers/plannerSlice.ts

```diff
@@ -265,6 +265,23 @@ const plannerSlice = createSlice({
         state.courses[code].mark = mark;
       }
     },
+    updateLegacyOfferings: (
+      state,
+      action: PayloadAction<{ code: string; offerings: CourseLegacyOfferings }>
+    ) => {
+      const { code, offerings } = action.payload;
+
+      if (state.courses[code]) {
+        if (state.courses[code].legacyOfferings === undefined) {
+          state.courses[code].legacyOfferings = {};
+        }
+
+        Object.entries(offerings).forEach(([year, terms]) => {
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
avoid doing eslint disable next line?

Reply:
I think it has to be there or I get warnings? I think the warnings are wrong tho because I initialize that value 2 lines above. Thoughts?